modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry_picker.jspf

```diff
@@ -115,6 +115,16 @@ DLFilePicker dlFilePicker = dlEditFileEntryDisplayContext.getDLFilePicker(lifera
 					'#<portlet:namespace />fm .lfr-ddm-form-container .ddm-field'
 				);
 
+				var urlTitleElement = document.getElementById(
+					'<portlet:namespace />urlTitle'
+				);
+
+				if (titleElement && titleElement.value) {
+					if (urlTitleElement && !urlTitleElement.value) {
+						urlTitleElement.value = titleElement.value.replace(/\.[^.]*$/, '');
+					}
+				}
```

Comment:
Can you merge these two conditional expressions? You can use a single if statement, there's no need for nesting here.